src/sentry/discover/models.py

```diff
@@ -40,10 +41,18 @@ class DiscoverSavedQuery(Model):
     date_updated = models.DateTimeField(auto_now=True)
     visits = BoundedBigIntegerField(null=True, default=1)
     last_visited = models.DateTimeField(null=True, default=timezone.now)
+    is_default = models.BooleanField(null=True, blank=True)
 
     class Meta:
         app_label = "sentry"
         db_table = "sentry_discoversavedquery"
+        constraints = [
+            UniqueConstraint(
+                fields=["organization", "is_default"],
+                condition=Q(is_default=True),
+                name="unique_user_default_query",
+            )
+        ]
```

Comment:
Should this index include user_id?

Reply:
Thanks for catching that. This should contain `created_by`. I've added it and regenerated the migration to include the change.